eeps/eep-0058.md

```diff
@@ -1,8 +1,8 @@
     Author: Sergey Prokhorov <seriy(dot)pr(at)gmail(dot)com>
-    Status: Final/26.0 Implemented in OTP release 26.0
+    Status: Final/26.0 Implemented in OTP release 26
     Type: Standards Track
     Created: 14-Sep-2021
-    Erlang-Version: 25
+    Erlang-Version: OTP-26.0
```

Comment:
I guess this was a typo in the EEP?

Reply:
No, not really. It was the Erlang version that the EEP was originally target at, but then it was released in a later version.